clients/ts/FunctionalTests/ts/WebSocketTests.ts

```diff
@@ -1,33 +1,37 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+import { WebSocketWrapper } from "@aspnet/signalr/dist/esm/Polyfills";
```

Comment:
Remove this and use `require` below (see comments below). We don't ever want to run `require('w3cwebsocket')` in  a browser environment.